sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/Config/SignalRFunctionsHostBuilderExtensions.cs

```diff
@@ -21,15 +21,14 @@ public static class SignalRFunctionsHostBuilderExtensions
         /// <summary>
         /// Adds security token validation parameters' configuration and SignalR connection's configuration.
         /// </summary>
-        /// <param name="builder">Azure function host builder</param>
+        /// <param name="services"></param>
```

Comment:
nit:  Please fill this out; this is a public member in a public class.

Reply:
Updated.